### ISSUE TITLE
[Bug]: Subscribing to non-Lemmy communities not working

### ISSUE THREAD

**Issue by BentiGorlich** [LemmyNet/lemmy#4299]
### Requirements

- [X] Is this a bug report? For questions or discussions use https://lemmy.ml/c/lemmy_support
- [X] Did you check to see if this issue already exists?
- [X] Is this only a single bug? Do not put multiple bugs in one issue.
- [X] Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.

### Summary

As far as I can tell subscribing to any non-Lemmy `Group` (AP term) is broken. It stays on "subscription pending" forever. I did not see any follow request activity getting sent in the logs

### Steps to Reproduce

1. Open Lemmy
2. Open any mBin magazine like !firefox@fedia.io
3. Subscribe to it
4. It stays on "subscription pending" for ever

### Technical Details

I ran lemmy in docker in v0.18.5. Since the message I was looking for was not in the log, I cannot post anything here. There were no errors in my log at all.

I cannot provide any logs from feddit.de since I am not running it

### Version

BE: 0.19.0, BE: 0.18.5

### Lemmy Instance URL

feddit.de and a local test instance

**Comment by BentiGorlich** [comment]
related mBin issue is here: mBinOrg/mbin#351

**Comment by BentiGorlich** [comment]
So I did a bunch of testing and what I found is that sometimes a lemmy test instance does not send a `Follow` activity. When it does it is working no problem (mBin always returns the `Accept` activity)...
As far as I can tell feddit has this problem of not sending a `Follow` activity, maybe try to get in touch with them to find out what is going on? Maybe its the general `0.19` federation problem, I don't know. However I am pretty sure the error is not on the mBin side
